diff --git a/strace-4.6/cde.c b/strace-4.6/cde.c
index 99edd2d..66b8fb3 100644
--- a/strace-4.6/cde.c
+++ b/strace-4.6/cde.c
@@ -1162,1 +1162,2 @@ void CDE_begin_at_fileop(struct tcb* tcp, const char* syscall_name) {
-  if (!IS_ABSPATH(filename) && tcp->u_arg[0] != AT_FDCWD) {
+  // See https://github.com/depaul-dice/provenance-to-use/pull/5
+  if (!IS_ABSPATH(filename) && ((int) tcp->u_arg[0]) != AT_FDCWD) {
@@ -2170,1 +2170,2 @@ void CDE_begin_file_unlinkat(struct tcb* tcp) {
-  if (!IS_ABSPATH(filename) && tcp->u_arg[0] != AT_FDCWD) {
+  // See https://github.com/depaul-dice/provenance-to-use/pull/5
+  if (!IS_ABSPATH(filename) && ((int) tcp->u_arg[0]) != AT_FDCWD) {
@@ -2232,1 +2232,2 @@ void CDE_begin_file_linkat(struct tcb* tcp) {
-  if (!IS_ABSPATH(oldpath) && tcp->u_arg[0] != AT_FDCWD) {
+  // See https://github.com/depaul-dice/provenance-to-use/pull/5
+  if (!IS_ABSPATH(oldpath) && ((int) tcp->u_arg[0]) != AT_FDCWD) {
@@ -2238,1 +2238,2 @@ void CDE_begin_file_linkat(struct tcb* tcp) {
-  if (!IS_ABSPATH(newpath) && tcp->u_arg[2] != AT_FDCWD) {
+  // See https://github.com/depaul-dice/provenance-to-use/pull/5
+  if (!IS_ABSPATH(newpath) && ((int) tcp->u_arg[2]) != AT_FDCWD) {
@@ -2307,1 +2307,2 @@ void CDE_begin_file_symlinkat(struct tcb* tcp) {
-  if (!IS_ABSPATH(newpath) && tcp->u_arg[1] != AT_FDCWD) {
+  // See https://github.com/depaul-dice/provenance-to-use/pull/5
+  if (!IS_ABSPATH(newpath) && ((int) tcp->u_arg[1]) != AT_FDCWD) {
@@ -2379,1 +2379,2 @@ void CDE_begin_file_renameat(struct tcb* tcp) {
-  if (!IS_ABSPATH(oldpath) && tcp->u_arg[0] != AT_FDCWD) {
+  // See https://github.com/depaul-dice/provenance-to-use/pull/5
+  if (!IS_ABSPATH(oldpath) && ((int) tcp->u_arg[0]) != AT_FDCWD) {
@@ -2385,1 +2385,2 @@ void CDE_begin_file_renameat(struct tcb* tcp) {
-  if (!IS_ABSPATH(newpath) && tcp->u_arg[2] != AT_FDCWD) {
+  // See https://github.com/depaul-dice/provenance-to-use/pull/5
+  if (!IS_ABSPATH(newpath) && ((int) tcp->u_arg[2]) != AT_FDCWD) {
