diff --git a/crt/crt1.c b/crt/crt1.c
index 10601215..1a42586f 100644
--- a/crt/crt1.c
+++ b/crt/crt1.c
@@ -5,7 +5,7 @@
 
 #include "crt_arch.h"
 
-int main();
+int main2() { return 0; }
 weak void _init();
 weak void _fini();
 int __libc_start_main(int (*)(), int, char **,
@@ -15,5 +15,5 @@ hidden void _start_c(long *p)
 {
 	int argc = p[0];
 	char **argv = (void *)(p+1);
-	__libc_start_main(main, argc, argv, _init, _fini, 0);
+	__libc_start_main(main2, argc, argv, _init, _fini, 0);
 }
diff --git a/src/env/__libc_start_main.c b/src/env/__libc_start_main.c
index c5b277bd..ed335945 100644
--- a/src/env/__libc_start_main.c
+++ b/src/env/__libc_start_main.c
@@ -67,11 +67,14 @@ static void libc_start_init(void)
 weak_alias(libc_start_init, __libc_start_init);
 
 typedef int lsm2_fn(int (*)(int,char **,char **), int, char **);
-static lsm2_fn libc_start_main_stage2;
+lsm2_fn libc_start_main_stage2;
 
 int __libc_start_main(int (*main)(int,char **,char **), int argc, char **argv,
 	void (*init_dummy)(), void(*fini_dummy)(), void(*ldso_dummy)())
 {
+	char helpme[] = "HELP, i'm stuck in a libc factory!";
+	__syscall(SYS_write, STDERR_FILENO, helpme, sizeof(helpme));
+
 	char **envp = argv+argc+1;
 
 	/* External linkage, and explicit noinline attribute if available,
@@ -86,12 +89,11 @@ int __libc_start_main(int (*main)(int,char **,char **), int argc, char **argv,
 	return stage2(main, argc, argv);
 }
 
-static int libc_start_main_stage2(int (*main)(int,char **,char **), int argc, char **argv)
+int libc_start_main_stage2(int (*main)(int,char **,char **), int argc, char **argv)
 {
 	char **envp = argv+argc+1;
 	__libc_start_init();
 
 	/* Pass control to the application */
-	exit(main(argc, argv, envp));
 	return 0;
 }
