diff --git a/stub_uploader/build_wheel.py b/stub_uploader/build_wheel.py
index 80d8ee45..762fb678 100644
--- a/stub_uploader/build_wheel.py
+++ b/stub_uploader/build_wheel.py
@@ -349,7 +349,7 @@ def generate_pyproject_file(
 ) -> str:
     """Auto-generate a setup.py file for given distribution using a template."""
     all_requirements = [
-        str(req) for req in metadata.requires_typeshed + metadata.requires_external
+        str(req) for req in metadata.requires_typeshed
     ]
     requires_python = (
         metadata.requires_python
